### ISSUE TITLE
Crash while training on Windows

### ISSUE THREAD

**Issue by ethanwhite** [weecology/deepforestr#9]
To replicate:

```r
library(deepforestr)

model = df_model()
model$use_release()

annotations_file = get_data("testfile_deepforest.csv")

model$config$epochs = 1
model$config["save-snapshot"] = FALSE
model$config$train$csv_file = annotations_file
model$config$train$root_dir = get_data(".")
model$config$train$fast_dev_run = TRUE

model$create_trainer()
model$trainer$fit(model)
```

In RGui this results in the following:
* Warning: Unknown option "-c"
* Warning: Unknown option "--multiprocessing-fork"
* Opening a new instance of RGui
* The main version of RGui hangs

* Possible route forward https://github.com/vim/vim/issues/906

**Comment by ethanwhite** [comment]
Looks like this is the same issue as #4 but it presents differently in RGui

**Comment by ethanwhite** [comment]
I believe this has now been fixed upstream (see #4), but RGui is generally a bit finicky with `reticulate` so we should spend some time testing and maybe explicitly document any specific requirements for manually setting the reticulate environment in RGui.

**Comment by ethanwhite** [comment]
We now have Windows testing including training on GH Actions (see #10). [Everything is working](https://github.com/weecology/deepforestr/actions/runs/9045907059/job/24856228901) so closing this, but if anyone runs into this issue on their local system feel free to reopen.
